internal/gcs/gcs_utils_test.go

```diff
@@ -0,0 +1,13 @@
+package gcs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUtils(t *testing.T) {
+	assert := assert.New(t)
+
+	assert.Equal("someFileName.json", bucket_path("someFileName.json"))
+}
```

Comment:
we probably should have a test that adds a prefix too.

Reply:
Done.